examples/bri-3/src/bri/merkleTree/models/bpiMerkleTree.ts

```diff
@@ -0,0 +1,22 @@
+import { AutoMap } from '@automapper/classes';
+import MerkleTree from 'merkletreejs';
+
+export class BpiMerkleTree {
+  @AutoMap()
+  id: string;
+
+  @AutoMap()
+  merkleTree: MerkleTree;
+
+  constructor(id: string, merkleTree: MerkleTree) {
+    this.id = id;
+    this.merkleTree = merkleTree;
+  }
+
+  public updateMerkleTree(
```

Comment:
consider moving in the hashing functionality to the domain object as well, as it is used only in the context of creating this object.